mmv1/third_party/terraform/website/docs/d/sql_database_instances.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "Cloud SQL"
+page_title: "Google: google_sql_database_instances"
+description: |-
+  Get a list of SQL database instances in a project in Google Cloud SQL.
+---
+
+# google\_sql\_database\_instance
+
+Use this data source to get information about a list of Cloud SQL instances in a project. You can also apply some filters over this list to get a more filtered list of Cloud SQL instances.
+
+## Example Usage
+
+
+```hcl
+data "google_sql_database_instances" "qa" {
+  project = "test-project"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (optional) The ID of the project in which the resource belongs. If it is not provided, the provider project is used.
+
+* `database_version` - (optional) To filter out the Cloud SQL instances which are of the specified database version.
+
+* `region` - (optional) To filter out the Cloud SQL instances which are located in the specified region.
+
+* `zone` - (optional) To filter out the Cloud SQL instances which are located in the specified zone.
```

Comment:
Let's check if this is same as location_preference.zone or not and update the documentation here.

Reply:
I have updated the documentation to add that clarity.